### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by galleon** [junzis/openap#28] [General]
Hi @junzis!

May I understand the reason for going for GPL license? I am part of a group working on `scikit-decide` a platform for planning and scheduling and it would make sense to have an example working with OpenAP but your license is contaminating. Our license is `MIT`.
At least, `LGPL` would be more friendly for us.

Cheers
Guillaume

**Comment by junzis** [comment]
Hi @galleon, yes, LGPL is indeed an option. I will try to change the license for future releases. You can count on that.

**Comment by junzis** [comment]
FYI, it is LGPL now :tada:
